rules/_deprecated/impact_potential_linux_ransomware_file_encryption.toml

```diff
@@ -1,8 +1,9 @@
 [metadata]
 creation_date = "2023/03/20"
+deprecation_date = "2024/07/18"
 integration = ["endpoint"]
-maturity = "production"
-updated_date = "2024/05/21"
+maturity = "deprecated"
```

Comment:
Optional: We could do the name change first, and deprecate in the next one

Reply:
Was thinking of the same thing, but this rule is bad in terms of performance and FPs, and I got several complaints about it before. So I wanted to get it out of the stack ASAP. We still don't have a guideline for this. Decided on just getting it out in one go, as many customers don't update pre-built rules either.